cvpy/annotation/cvat/CVATProject.py

```diff
@@ -70,28 +81,72 @@ def _delete_project_in_cvat(self) -> None:
         if response.status_code != HTTPStatus.NO_CONTENT:
             raise Exception(f'Unable to delete the project in the CVAT server: {response.reason}')
 
-    def post_images(self, image_table: CASTable) -> None:
+    def post_images(self, image_table: ImageTable) -> None:
         '''
         Create a CVAT task under the project and upload images from a CAS table to that task.
 
         Parameters
         ----------
         image_table: 
-            Specifies the input CAS table that contains encoded images to be uploaded.
+            Specifies the input table that contains encoded images to be uploaded.
         
         '''
-        pass
+        # Check that the image_table contains the required columns for posting images to CVAT.
+        if image_table.id == None:
+            raise Exception(f'Provided ImageTable is missing a required column: id')
+        elif image_table.image == None: 
+            raise Exception(f'Provided ImageTable is missing a required column: image')
+        elif image_table.type == None:
+            raise Exception(f'Provided ImageTable is missing a required column: type')
+
+        # Fetch the image table that will be posted to CVAT.
+        image_table_fetched = image_table.table.fetchImages(fetchImagesVars=["_id_", "_type_"])
+
+        # Create a CVATTask for this set of images.
+        task = CVATTask(str(0), str(image_table_fetched['Images'].shape[0]-1), image_table, self)
+
+        # Build a dictionary of images to upload to CVAT.
```

Comment:
Can you please add a comment stating what is the key and what is the value in this dictionary?